massgen/tests/test_validate_all_configs_script.py

```
@@ -0,0 +1,64 @@
+    config = {
+        "agent": {
+            "id": "agent-1",
+            "backend": {"type": "openai", "model": "gpt-4o"},
+        },
+    }
+    config.update(config_extra)
+    config_path.write_text(
+        yaml.safe_dump(config),
+    )
+
+    result = subprocess.run(
```

Ruff (0.15.14)

[error] `subprocess` call: check for execution of untrusted input

(S603)